README.md

```diff
@@ -48,13 +48,42 @@ Git Proxy is built with a developer-first mindset. By presenting simple-to-follo
 To install Git Proxy, use the [npm](https://www.npmjs.com/) package manager:
 
 ```bash
-$ npm install @finos/git-proxy
+$ npm install -g @finos/git-proxy
```

Comment:
@coopernetes - why `-g`?

Reply:
Since we’re providing an executable, installing via `-g` will put it on the user’s PATH. Without `-g`, the runnable server would only be available in a local (current working directory) `node_modules/`. I can investigate if there is a middle ground which will install this to something like `$HOME/.local/bin`.

Reply:
npx instructions are now provided. @JamieSlome - can we resolve this conversation?